src/components/Footer.tsx

```diff
@@ -0,0 +1,100 @@
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { TodosContext } from '../stores/TodosContext';
+import { FilterParams } from '../types/FilterParams';
+
+type Props = {
+  filterBy: FilterParams;
+  setFilterBy: (v: FilterParams) => void;
+};
+
+export const Footer: React.FC<Props> = ({ filterBy, setFilterBy }) => {
+  const {
+    deleteTodo,
+    todos,
+    unCompletedTodos,
+    completedTodos,
+  } = useContext(TodosContext);
+
+  const todosCounter = unCompletedTodos.length;
+
+  const handleAllClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault();
+    setFilterBy(FilterParams.All);
+  };
+
+  const handleActiveClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault();
+    setFilterBy(FilterParams.Active);
+  };
+
+  const handleCompletedClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault();
+    setFilterBy(FilterParams.Completed);
+  };
```

Comment:
```suggestion
  const handleFilter = (filter: FilterParams) => (e: React.MouseEvent<HTMLAnchorElement>) => {
    e.preventDefault();
    setFilterBy(filter);
  };
```
better to make one function factory and use it for all filter buttons